axelrod/strategies/momentum.py

```diff
@@ -0,0 +1,58 @@
+from axelrod.action import Action
+from axelrod.player import Player
+
+C, D = Action.C, Action.D
+
+
+class Momentum(Player):
+    """
+    This strategy is inspired by the concept of Gradual and the mathematical foundation of
+    the Momentum optimizer used in deep learning.
+
+    The idea is that trust (or cooperation) evolves dynamically. A shift in trust can
+    create significant and rapid changes in the player's behavior, much like how momentum
+    responds to gradients in optimization.
+
+    Names:
```

Comment:
Could you add a few sentences about the role of the alpha, momentum and the threshold here please.

Then you could add a quick description of their default values which means we can remove the comments in the `init` method.